src/circuits/getsamples.jl

```diff
@@ -6,6 +6,29 @@
 --------------------------------------------------------------------------------
 """
 
+function fullbases(N::Int; local_basis = "Pauli")
```

Comment:
Added a set of informationally complete bases, which scales exponentially with n (for few-qubit tomography).